code/modules/xenobio/items/extracts.dm

```diff
@@ -564,12 +564,19 @@
 		if(S.stat || S.docile || S.rabid)
 			continue
 
+		S.add_modifier(/datum/modifier/berserk, 30 SECONDS)
+
 		if(S.client) // Player slimes always have free will.
-			to_chat(S, "<span class='warning'>An intense wave of rage almost overcomes you, but you remain in control of yourself.</span>")
+			to_chat(S, "<span class='warning'>An intense wave of rage is felt from inside, but you remain in control of yourself.</span>")
 			continue
 
 		S.enrage()
 
+	for(var/mob/living/carbon/human/H in view(get_turf(holder.my_atom)))
+		if(H.species.name == "Promethean")
+			H.add_modifier(/datum/modifier/berserk, 30 SECONDS)
+			to_chat(H, "<span class='warning'>An intense wave of rage is felt from inside, but you remain in control of yourself.</span>")
```

Comment:
It says they remain in control, but they've got the same Berserk modifier, is that intended?

Reply:
Meant to show that they still have free will and such and don't go 'rabid' as their slime counterparts might.

Reply:
Alrighty.